src/main/java/eu/greev/dcbot/ticketsystem/service/TicketService.java

```diff
@@ -255,7 +289,9 @@ public Ticket getTicketByChannelId(long idLong) {
 
         return optionalTicket.orElseGet(() -> {
             Ticket loadedTicket = ticketData.loadTicket(idLong);
-            if (loadedTicket.getOwner() == null) return null;
+            if (loadedTicket.getOwner() == null) {
```

Comment:
Not sure if this could result in a problem but what happens if loadedTicket is null

Reply:
There should be only one case where this method returns null and this one is secured at the usage place